### PR TITLE
Intro NSM and Cilium

### PR DESCRIPTION
**Pull request by wiggitywhitney**

### CODE REVIEW

File: manuscript/mtls/README.md [hunk 1]
```diff
@@ -16,19 +16,19 @@ Network Policies and mTLS are often added to a Kubernetes cluster using an appli
 
 ## Choice 1: Istio
 
-Like [Ingress](manuscript/ingress/README.md), the L7 service mesh technologies (Linkerd, Istio, Kuma) each act as a *control plane* that does the complicated work of configuring and managing an underlying networking technology (often referred to as the *data plane*). This data plane can also be called a *networking layer*, *service proxy*, or just *proxy*. The job of the data plane is to add security, capture usage data, and, most importantly, actually route the requests.
+Like [Ingress](manuscript/ingress/README.md) technologies, the L7 service mesh technologies (Linkerd, Istio, Kuma) each act as a *control plane* that does the complicated work of configuring and managing an underlying networking technology (often referred to as the *data plane*). This data plane can also be called a *networking layer*, *service proxy*, or just *proxy*. The job of the data plane is to add security, capture usage data, and, most importantly, actually route the requests.
```
Comment (vfarcic) [inline manuscript/mtls/README.md:19]: This and the next paragraph are generic and not necessarily applicable only to Istio (it even mentions others). Would it make sense to move it up to the intro?

File: manuscript/mtls/README.md [hunk 2]
```diff
@@ -16,19 +16,19 @@ Network Policies and mTLS are often added to a Kubernetes cluster using an appli
...
+It is most efficient to run Istio in Kubernetes, but workloads can run anywhere. Istio can also support VMs as part of the mesh, as well as provide multi-cluster mesh. Istio has a sidecarless model called Ambient Mesh. 
 
 
 [![Sailing Into a Sea of Microservices With Istio](https://img.youtube.com/vi/W3K2Ufng8Zg/0.jpg)](https://youtu.be/W3K2Ufng8Zg)
 * [Official site](https://istio.io)
 
-## Choice 2: Linkerd (SMI)
+## Choice 2: Linkerd
 
 To recap: *service mesh* is an infrastructure layer that adds security, networking, and observability uniformly across your entire application and, when done well, across your entire system. These characteristics are not optional; if they’re not added via service mesh, developers have to add them directly to their applications.
```
Comment (vfarcic) [inline manuscript/mtls/README.md:33]: This also sounds like it's not specific to LinkerD. Move to the intro?

File: manuscript/mtls/README.md (hunk 1 above)
Comment (wiggitywhitney) [reply manuscript/mtls/README.md:19]: I hear what you're saying. The statements are about the app layer meshes so they're in between - they apply to 3/5 of the technologies in this section.

I'm not sure that I want to open that can of worms in the intro, especially when it doesn't apply to everything. Maybe we have a separate intro for the L7 meshes. Let's talk it out during preshow today.
